packages/apollo-links/src/core/clusterAuthLink.ts

```diff
@@ -79,19 +79,21 @@ export class ClusterAuthLink extends ApolloLink {
     const type = OrderType.agreement;
     const { token, id, url, indexer } = nextAgreement;
     if (!isTokenExpired(token)) return { token, url, type };
+
     this.logger.debug(`request new token for indexer ${indexer}`);
-    const { projectId, authUrl } = this.options;
 
+    const { projectId, authUrl } = this.options;
     const tokenUrl = new URL('/orders/token', authUrl);
     const res = await POST<{ token: string }>(tokenUrl.toString(), {
       projectId,
       indexer,
       agreementId: id,
     });
 
-    this.orderMananger.updateTokenById(id, res.token);
+    const updatedToken = `Bearer ${res.token}`;
+    this.orderMananger.updateTokenById(id, updatedToken);
     this.logger.debug(`request new token for indexer ${indexer} success`);
-    return { token: `Bearer ${res.token}`, url, type };
```

Comment:
should we stick to the concept: 
* `token` <=> `token get from auth service`
* `autherization` <=> `Bearer ${token}`
It will avoid a lot of errors in the future caused by the confusion

Reply:
Make sense, can change token use in this link to `autherization`